test/Altinn.AccessManagement.Tests/Controllers/RightsInternalControllerTest.cs

```diff
@@ -50,7 +50,7 @@ public class RightsInternalControllerTest : IClassFixture<CustomWebApplicationFa
         /// <param name="factory">CustomWebApplicationFactory</param>
         public RightsInternalControllerTest(CustomWebApplicationFactory<RightsInternalController> factory)
         {
-            _factory = factory;
+            fixture = factory;
```

Comment:
Rename factory to fixture for consistency?

Reply:
done